debian/rules

```diff
@@ -18,14 +21,14 @@ override_dh_installman:
 override_dh_installchangelogs:
 	dh_installchangelogs NEWS upstream
 
-## TODO: build currently is not done during package creation.
-override_dh_autoreconf:
-	true
-
-## TODO: build currently is not done during package creation.
-override_dh_auto_configure:
-	true
-
-## TODO: build currently is not done during package creation.
-override_dh_auto_build:
-	true
+# ## TODO: build currently is not done during package creation.
+# override_dh_autoreconf:
+# 	true
+#
+# ## TODO: build currently is not done during package creation.
+# override_dh_auto_configure:
+# 	true
+#
+# ## TODO: build currently is not done during package creation.
+# override_dh_auto_build:
+# 	true
```

Comment:
What's the point in merely commenting this out? Maybe we should invoke the actual build from here?

Reply:
I am not sure yet we are going for compilation during package build due to https://github.com/ResistancePlatform/resistance-miner/issues/12 and due to other platform builds, as well as other package builds (such as https://github.com/ResistancePlatform/resistance-core/issues/71).

Since comments don't do harm, I left them in. Once things are figured out / stablized it's easy to wipe the comments.